docs/tutorials/processing/dask.md

```diff
@@ -205,7 +207,7 @@ directory. The dataset is 3.69 GB unzipped.
 ```
 
 While we could use plain `dask` through `delayed`, as we have shown above, we
-will [make our lives easier](make-your-life-easier-with-dask-image) here and use
+will [make our lives easier](dask_images) here and use
```

Comment:
Perfectly fine to create a new target, but FYI if we do `[make our lives easier](#make-your-life-easier-with-dask-image)` the automatic label works.